### ISSUE TITLE
Better support multi account 

### ISSUE THREAD

**Issue by simcap** [wallix/awless#166]
When using `awless` and moving between AWS accounts (that can be referenced by AWS profiles) to work between different infrastructure, the local cloud data are being overwritten only on the first sync.

This can cause issues such as: changing profile and auto completion returning resources from another account (i.e. profile) because a sync as not been done yet (autocompletion works with local synced data); etc.

(referencing @padilo originator of this issue)

**Comment by simcap** [comment]
Indeed, the fact that multi-account is not fully supported can causes issues. This has been on our roadmap for some time and should be completed in the next few months. That is paramount for a smooth usage of awless for people with multi-account

I have personally already tried a implementation to store the profile/account info as part of the local data file tree (under ~/.awless/aws/rdf/...). It did not work and scale well. We will give that another go and come up with some other solutions. 

For instance, we might introduce a `awless switch` command, etc. Any ideas, solutions is welcome.

**Comment by varunchandak** [comment]
Even I am using multi-accounts with approx 7-8 and it's a pain to sync everytime for each account.

**Comment by padilo** [comment]
About having `awless switch` I'm a bit against that, because I prefer to explicitly specify the account I'm working on, to be sure where I'm doing things.

**Comment by simcap** [comment]
Indeed `awless switch` is not the multi account solution ;) , but merely a convenience with completion to switch to a region/profile (simultaneously even) instead of having each time the cumbersome `awless config set aws.region ....`.

Anyway, the good news is that we are going to be focusing on the multi account in the 2 coming weeks and we will provide way of specifying clearly the account. What we want to avoid in a CLI is confusion!

Stay tuned!

**Comment by simcap** [comment]
@padilo @varunchandak Multi account (i.e. taking profile into account when syncing) is now supported in master. 

It will be released soon in the next v0.1.9. You can already use it by pulling master (or go get -u .... then `go install`).

Let us know how that works for you. Thanks!

**Comment by simcap** [comment]
@padilo @varunchandak Note that there is nothing to be done on your side ;)

Basically an account is represented by a profile name. So you just use the `awless switch` to change your AWS profile and region (i.e. account) and the sync will work seamlessly.
